OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -529,21 +495,28 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
         _slaveBodyForChild = slaveForChild;
     }
 
-private:
-    //=========================================================================
+    //==========================================================================
     // DATA
-    //=========================================================================
+    //==========================================================================
+protected:
+    // Specifies the direction of the Joint in the multibody tree: parent->child
+    // (isReversed is false; default) or child->parent (isReversed is true). The
+    // Joint's transform and coordinates maintain a parent->child sense even if
+    // the joint is reversed.
+    bool isReversed = false;
```

Comment:
Addressed feedback:
- Added "Remove `Joint::getIsReversed()`" to "To be addressed in part 2" list
- Added note to CHANGELOG